docs/_config_dev_en.yml

```diff
@@ -17,3 +17,6 @@
 
 exclude:
   - "*.zh.md"
+  - "content"
```

Comment:
are these required for correctness?

Reply:
if not excluded, they may end up in the generated folder (as yet another subdirectory) again, e.g. when the zh version builds, if would not see the `content_en` as a target dir (and exclude it)